project/StacksafeScalapbGenerator.scala

```diff
@@ -132,10 +132,16 @@ class StacksafeProtobufGenerator(params: GeneratorParams) extends ProtobufGenera
     super
       .generateSerializedSize(message)(withEqualsAndHashCode)
       .newline
-      .add(
-        "@transient val serializedSizeM = new coop.rchain.models.Memo(coop.rchain.models.ProtoM.serializedSize(this))"
-      )
+      .add("@transient var _serializedSizeM: coop.rchain.models.Memo[Int] = null")
```

Comment:
I think you've meant `@volatile` here

Reply:
it was transient before, i though it was for a reason (when being serialized). you tell me, you wrote it.

Reply:
ah, right - it's needed so that this doesn't get serialized. :+1: So it should be `@transient`, but not `@volatile` given my latter comment (let's use synchronized)